CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+6. Provide your full name or GitHub username and your company organization if applicable.
+
+#### Once a feature request is submitted:
+1. The project team will label the issue.
+2. The project team will evaluate the feature request, possibly asking you more questions to understand its purpose and any relevant requirements. If the issue is closed, the team will convey their reasoning and suggest an alternative path forward.
+3. If the feature request is accepted, it will be marked for implementation.
+
+### Security Vulnerabilities
+Please view our [Security Policy](https://github.com/nasa/cFS/security/policy) for more information.
+
+### Pull Requests
+#### Ready to Add Your Code?
+
+Follow GitHub's fork-branch-pull request pattern.
+1. Fork the relevant cFS component (eg. cfe, osal, psp).
+2. Create a new branch in your fork to work on your fix. We recommend naming your branch `fix-ISSUE_NUMBER-<FIX_SUMMARY>`.
+3. Submit a pull request to the nasa `main` branch. We recommend creating your pull-request as a "draft" and to commit early and often so the community can give you feedback at the beginning of the process as opposed to asking you to change hours of hard work at the end.
+4. Add commits to your branch. Please follow commit message convention: `Fix ISSUE_NUMBER, 50-ish-character-long summary of commit content`.
+5. Sign and email the appropriate Contributor License agreement down below to GSFC-SoftwareRelease@mail.nasa.gov and copy cfs-program@lists.nasa.gov.
+    - Corporate Contributor License agreement : https://github.com/nasa/cFE/blob/main/docs/GSC_18128_Corp_CLA_form_1219.pdf
+    - Individual Contributor License agreement : https://github.com/nasa/cFE/blob/main/docs/GSC_18128_Ind_CLA_form_1219.pdf
+
+#### Create a Pull Request
+1. For the title, use the title convention `Fix #XYZ, SHORT_DESCRIPTION`.
+2. Describe the contribution. First document which issue number was fixed using the template "Fix #XYZ". Then describe the contribution.
+3. Provide what testing was used to confirm the pull request resolves the link issue.
+4. Provide the expected behavior changes of the pull request.
+5. Provide the system the bug was observed on including the hardware, operating system, and versions.
+6. Provide any additional context if applicable.
+7. Provide your full name or GitHub username and your company or organization if applicable.
+
+#### Once a Pull Request Ready for Review
+1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`.
+2. Verify there is only one commit message. Squash or amend the commit messages if necessary.
+3. Verify that the PR passes all checks.
+4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process.](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process)
+5. If the pull request is accepted, it will be merged into cFS.
+
+### Discussions and Questions
```

Comment:
Maybe we should move this up so people see it earlier. Also we should distinguish between q&a community type things which should go to https://lists.nasa.gov/mailman/listinfo/cfs-community versus programmatic questions like requests for information, partnerships, training, etc.

Reply:
Moved Discussions and Questions as the first section of Ways to Contribute.